src/server/game/Spells/Spell.cpp

```diff
@@ -3457,7 +3458,7 @@ void Spell::DoProcessTargetContainer(Container& targetContainer)
             if (target.EffectMask & (1 << i))
                 target.DoTargetSpellHit(this, i);
 
-    for (TargetInfoBase& target : targetContainer)
+    for (uint8 i = 0; i < MAX_SPELL_EFFECTS; ++i)
```

Comment:
this is definitely not right (does it even build?)

Reply:
no, working on amending it, forgot to compile first